python/spglib/spglib.py

```diff
@@ -97,6 +111,88 @@ class SpglibError:
 spglib_error = SpglibError()
 
 
+@dataclasses.dataclass(eq=True, frozen=True)
+class DictInterface(Mapping[str, "Any"]):
```

Comment:
Did you find that`@dataclass` was needed for this?

Reply:
Yeah, needed for mypy